validator.go

```diff
@@ -467,11 +470,18 @@ func parseName(tag string) string {
 		return ""
 	}
 
-	name := strings.SplitN(tag, ",", 2)[0]
-
+	tags := strings.Split(tag, ",")
+	name := tags[0]
 	// if the field as be skipped in json, just return an empty string
 	if name == "-" {
 		return ""
 	}
+
+	for _, v := range tags {
+		if v == "inline" {
+			return ""
```

Comment:
This part seems unnecessary. If you specified `json:blah,inline` that's invalid and a linter should complain

Reply:
This would also break `json:"inline"`

Reply:
You are right. Thanks.

Also knew that
```
struct A {
  FieldName B `",inline"`
}
```
does not work in default go encoding; there is no special meaning for "inline" tag.